### PR TITLE
ISearch Interface implemented

### PR DESCRIPTION
**Pull request by aburgosu**

### CODE REVIEW

File: src/main/java/com/fundation/logic/model/ISearch.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ /*
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/model/ISearch.java]: One * missing

File: src/main/java/com/fundation/logic/model/ISearch.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  */
+
+ package com.fundation.logic.model;
+
+ /* Implements the model class File and the getter and setter´s methods
+  * @author Andres Burgos
+  * @version 1.0 *
+  */
+
+ interface ISearch{
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/model/ISearch.java]: One space missing after ISearch, before {

File: src/main/java/com/fundation/logic/model/ISearch.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  * This software is the confidential and proprietary information of Jalasoft.
+  * ("Confidential Information"). You shall not
+  * disclose such Confidential Information and shall use it only in
+  * accordance with the terms of the license agreement you entered into
+  * with Jalasoft.
+  */
+
+ package com.fundation.logic.model;
+
+ /* Implements the model class File and the getter and setter´s methods
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/model/ISearch.java]: One * missing after /*
Please look at the example below.
/**
 * Implements the VideoPlayerFrame class Frame
 *
 * @author John Salazar Pinto
 * @version 1.0
 */

File: src/main/java/com/fundation/logic/model/ISearch.java (hunk 1 above)
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/ISearch.java]: use /** for start a coment

File: src/main/java/com/fundation/logic/model/ISearch.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  * @author Andres Burgos
+  * @version 1.0 *
+  */
+
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/ISearch.java:22]: delete all spaces after a comment ends

**Pull request merged**